docker-for-mac/index.md

```diff
@@ -97,107 +99,90 @@ docker-machine version {{ site.machine_version }}, build 9ba6da9
     $ docker image rm nginx
     ```
 
-## Preferences menu
+## Preferences
 
-Choose ![whale menu](images/whale-x.png){: .inline}  → **Preferences** from the
+Choose the Docker menu ![whale menu](images/whale-x.png){: .inline} > **Preferences** from the
 menu bar and configure the runtime options described below.
 
 ![Docker context menu](images/menu/prefs.png){:width="250px"}
 
 ### General
```

Comment:
Are the settings the same for the Stable and Edge releases? Just curious because I am running the Stable release, and there are different options under General.